tests/nodeset-loader/testnodeset.xml

```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The test-nodeset could be hosted in the repo of the nodeset loader.
It is a good candidate to be used also in the nodeset loader repo directly.